### PR TITLE
feat(SkeletonButton): Add Skeleton Button

### PR DESCRIPTION
**Pull request by assuncaocharles**
#### Pull request checklist

- [ ] Addresses an existing issue: Fixes #0000
- [ ] Include a change request file using `$ yarn change`

#### Description of changes

This PR adds `Skeleton.Button` component to be used as `Button` placeholder while in loading state

Screenshot 2020-08-10 at 11 37 21

#### Focus areas to test

(optional)

### CODE REVIEW

**Comment by dzearing** [comment]
I don't know what a SkeletonButton is, and there's no description on the PR

File: packages/fluentui/react-northstar/src/components/Skeleton/Skeleton.tsx [hunk 1]
```diff
@@ -0,0 +1,53 @@
...
+   */
+  animation?: 'pulse' | 'wave';
+}
+export interface SkeletonProps extends SkeletonOwnProps, Omit<BoxProps, 'accessibility'> {}
+
+export type SkeletonStylesProps = Required<Pick<SkeletonProps, 'animation'>>;
+export const skeletonClassName = 'ui-skeleton';
+
+/**
+ * An Skeleton is a component to be placed while the content is loading.
```
Comment (dzearing) [inline packages/fluentui/react-northstar/src/components/Skeleton/Skeleton.tsx:27]:
```suggestion
 * A Skeleton is a component to be placed while the content is loading.
```

File: packages/fluentui/react-northstar/src/components/Skeleton/Skeleton.tsx [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+ */
+export const Skeleton = compose<'div', SkeletonOwnProps, SkeletonStylesProps, BoxProps, BoxStylesProps>(Box, {
```
Comment (dzearing) [inline packages/fluentui/react-northstar/src/components/Skeleton/Skeleton.tsx:29]: i'm confused. Is this a Shimmer?

**Comment by assuncaocharles** [comment]
> It would be good to add a PR description here :)

Hey @dzearing will add it once change the status from draft as this PR will be still follow up from #14354

File: packages/fluentui/docs/src/examples/components/Skeleton/Variations/SkeletonExampleComponents.tsx [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+      <Flex gap="gap.medium" column>
+        <Flex gap="gap.medium">
+          <Skeleton.Button />
+          <Skeleton.Button size="small" />
+          <Skeleton.Button circular />
+          <Skeleton.Button circular size="small" />
+          <Skeleton.Button iconOnly />
+          <Skeleton.Button iconOnly size="small" />
+        </Flex>
+        <div>
```
Comment (pompomon) [inline packages/fluentui/docs/src/examples/components/Skeleton/Variations/SkeletonExampleComponents.tsx:19]:
```suggestion
        <Flex.Item>
```
_nit_

Comment (assuncaocharles) [reply packages/fluentui/docs/src/examples/components/Skeleton/Variations/SkeletonExampleComponents.tsx:19]: Done

**Pull request merged**